internal/controlplane/handlers_profile.go

```diff
@@ -513,19 +524,17 @@ func (s *Server) getRuleEvalStatus(
 		if dbRuleEvalStat.EntityType == db.EntitiesRepository {
 			repoPath = fmt.Sprintf("%s/%s", st.EntityInfo["repo_owner"], st.EntityInfo["repo_name"])
 		} else if dbRuleEvalStat.EntityType == db.EntitiesArtifact {
-			repoDetails, err := s.store.GetRepoPathFromArtifactID(ctx, dbRuleEvalStat.EntityID)
-			if err != nil {
-				l.Err(err).Msg("error getting repo details from db")
-				return st
+			artRepoOwner := efp.Properties.GetProperty(ghprop.ArtifactPropertyRepoOwner).GetString()
+			artRepoName := efp.Properties.GetProperty(ghprop.ArtifactPropertyRepoName).GetString()
+			if artRepoOwner != "" && artRepoName != "" {
+				repoPath = fmt.Sprintf("%s/%s", artRepoOwner, artRepoName)
```

Comment:
Is this name ever going to be used in anything but presentation? I wonder if we should make the RepoOwner and RepoName properties not tied to github provider (and move them into contants.go like we have properties.RepoPropertyIsPrivate). Then we could use this code for non-github artifacts and maybe even make the code more generic by constructing new properties and calling provider.GetEntityName(REPOSITORY, theNewProperties)

Reply:
Unfortunately, our output is heavily tied to GitHub, this is actually used to get the alert which won't exist in other providers. I can't wait to deprecate that.

Reply:
Answering your question, I still think it's a github-specific attribute.